packages/cli/src/bin/cli.ts

```diff
@@ -58,12 +58,7 @@ program
     '--load-delay <n>',
     'Set how much time (milliseconds) axe will wait after page load before running the audit (default: 0)'
   )
-  .option(
-    '--timeout <n>',
-    'Set how much time (seconds) axe has to run',
-    // @ts-ignore
-    90
-  )
+  .option('--timeout <n>', 'Set how much time (seconds) axe has to run', '90')
```

Comment:
Do you have any idea why it was passed as a number previously? Seems like an intentional thing to me for whatever reason, especially since it was ts-ignored 🤔

Reply:
The [original pr](https://github.com/dequelabs/axe-core-npm/pull/683) has no useful context on that. Looks to me like they just knew it should be a number so they made it a number by force. When if you use `--timeout 120` in the terminal, it is still coming in as a string and needs to be accounted for. The math is casting correctly, so it works as a string.